.github/workflows/sam_release_code.yml

```diff
@@ -142,6 +145,7 @@ jobs:
           DOMAIN_NAME_EXPORT: ${{ inputs.DOMAIN_NAME_EXPORT }}
           ZONE_ID_EXPORT: ${{ inputs.ZONE_ID_EXPORT }}
           TOGGLE_GET_STATUS_UPDATES: ${{ inputs.TOGGLE_GET_STATUS_UPDATES }}
+          ENABLE_ALERTS: true
```

Comment:
this should use the value passed into the workflow, not a hard coded value

Reply:
totally forgot to remove that once I'd tested 🤦
c0119bf4fb5eee8a615df850f40f9ec6708691bc